pkg/kev/kev.go

```diff
@@ -123,3 +124,72 @@ func Render(format string, singleFile bool, dir string, envs []string) error {
 
 	return nil
 }
+
+// Dev continuously watches source compose files and rebuilds Kubernetes manifests on each change.
+// Note: It'll rebuild manifessts for all tracked environments, unless list of environments specifically provided.
+func Dev(envs []string, change chan<- string) error {
+	workDir, err := os.Getwd()
```

Comment:
It would be great if we can pass in the workDir as a param.

It makes testing easier and helps us use the func in a lib context.

Reply:
Watch func just takes a list of files to watch and a channel. Caller decides what needs to be watched.